stream/vibe/stream/wrapper.d

```diff
@@ -254,7 +258,8 @@ struct StreamInputRange {
 		Buffer* m_buffer;
 	}
 
-	this (InputStream stream)
+	/// private
+	this(InputStream stream)
```

Comment:
Why the `private` comment instead of actually being `private` ?

Reply:
Hm, I just copied it from the way `StreamOutputRange` does it, but at a second glance it looks like it could indeed just be properly `private`.